frontend/server/controllers/ProblemController.php

```diff
@@ -1366,10 +1366,18 @@ private static function validateProblemset(Problems $problem, $problemsetId, $co
      * @throws InvalidDatabaseOperationException
      */
     public static function apiDetails(Request $r) : array {
-        return self::getProblemDetails($r);
+        if (!isset($r['show_solvers'])) {
+            return self::getProblemDetails($r);
+        }
+        $r->ensureBool('show_solvers');
```

Comment:
```suggestion
        $r->ensureBool('show_solvers', /*required=*/false);
```

así no tienes que tener L1369-1371

Reply:
Para que me funcione, tengo que seguir agregando en la condición de L1536, que `$r['show_solvers]` exista y tanga valor `true`, o ¿De qué otra forma lo puedo hacer?

Reply:
ah puedes checar `$r['show_solvers'] === true` y pasarle eso como argumento a la función.